tools/build_vic3_to_hoi4.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+cmake -H. -Bbuild && cmake --build ./build -- -j40 && cd build/test/Release-Linux && ./Vic3ToHoI4Tests && cd ..
```

Comment:
Reason why we have two files, one local and other "foreign" is because github's action runners have 2 cores total. Swarming them with 40 threads might be tad overkill. 3 is good as it can shuffle them about. Our local runner on the other hand has 32 I think so 40 is a good target. See ck3toeu4 for example.

Reply:
I actually just run the commands directly in the action scripts rathe than calling the script. The script is for local use only.